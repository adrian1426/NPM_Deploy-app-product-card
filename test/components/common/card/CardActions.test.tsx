import React from 'react';
import renderer from 'react-test-renderer';
import CardActions from '../../../../src/components/common/card/CardActions';

describe('CardActions', () => {
  test('should render CardActions', () => {
    const wrapper = renderer.create(
      <CardActions />
    );

    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});