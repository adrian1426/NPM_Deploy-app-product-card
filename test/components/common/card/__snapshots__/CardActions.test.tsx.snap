// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardActions should render CardActions 1`] = `
<div
  className="buttonsContainer undefined"
/>
`;
